taker-frontend/src/components/ConfirmOrderModal.tsx

```diff
@@ -69,9 +73,10 @@ export default function ConfirmOrderModal({
                 <ModalHeader>
                     <HStack>
                         <Text>
-                            Market sell <b>{quantity}</b> of BTC/USD @
+                            Market {buy_or_sell}
```

Comment:
```suggestion
                            Market {buy_or_sell}&nbsp;
```

I think the linter / dprint removed that space for you - probably better to add an explicit one with `&nbsp;` otherwise it reads like this:

![image](https://user-images.githubusercontent.com/5557790/157171260-0276a1ba-64af-4999-a495-e5f495d373ce.png)

Reply:
Good catch